test/lib/PortMap.h

```diff
@@ -0,0 +1,84 @@
+// Track TCP ports to use with the tester.
+class PortMap {
+  public:
+
+    static const int64_t START_PORT = 10000;
+    static const int64_t MAX_PORT = 20000;
+
+    // Get an unused port
+    uint16_t getPort()
```

Comment:
since you moved this class to it's own file it makes sense to move these into their own implementation file, there's no reason they need to be in the header.